presto-docs/src/main/sphinx/release/release-0.280.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.280
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug where if a query was submitted before the resource group configuration had been loaded, all queries to the cluster would fail with a ``NullPointerException``. This fixes a race condition where a query can start constructing resource groups before the ``ResourceGroupConfigurationManager`` is loaded. 
+* Fix a bug in the ``GatherAndMergeWindows`` optimization which produces invalid plans when the output of a window function is used in the frame definition of another window function.
+* Fix output of :func:`find_first` function for ``NULL`` Now it will throw exception if the found matched value is NULL.
+* Fix the bug where duplicate items in ``UNNEST`` leads to query compilation failure.
+* Add :func:`remove_nulls` to remove null elements from the given array.
+* Add UDF `find_first_index` Returns the index of the first element which satisfies the condition.
+* Add range with offset expression support to [window function](https://prestodb.io/docs/current/functions/window.html?highlight=window#functions-window--page-root).
+* Add the names of the functions invoked by the query to the completed event. This can be enabled with the ``log_invoked_function_names_enabled`` session property or the ``log-invoked-function-names-enabled`` configuration property.
+* Added a new optimization for filtering large tables with ``LIMIT`` number of keys for queries that do simple ``GROUP BY LIMIT`` with no ``ORDER BY``. This feature can be enabled with a boolean session param: ``prefilter_for_groupby_limit``
+* Added a new runtime metric ``optimizerTimeNanos`` to measure time taken by optimizers. With this change the time taken by optimizers has been removed from runtime metric ``logicalPlannerTimeNanos``.
```

Comment:
`to measure time taken by optimizers` -> `to measure the time taken by the optimizers`